100-level-AsyncAPI-Intro/step1.md

```diff
@@ -0,0 +1,54 @@
+Eve assumes her problem-solving role and approaches the spaceship. Inside, Eve discovers Chan, a programming genius visiting from Brownieterra. It becomes apparent that a technical glitch has affected the ship's communication system.
+
+With her expertise in event-driven architecture (EDA) and AsyncAPI, Eve swiftly springs into action. She formulates a plan to establish seamless communication among the spaceship's components in collaboration with Chan.
+
+Eve begins by providing Chan with a comprehensive explanation of event-driven architecture (EDA) and outlines the process of designing an EDA system using AsyncAPI.
+
+## Event-driven architecture
+
+An event signifies a change in the system's state that triggers communication between different services. Event-driven architecture (EDA) decomposes an application into several components interacting via events. These components encompass event producers and consumers:
+
+- An event **producer** identifies an event and transmits relevant information through the event channel.
+- An event **consumer** subscribes to an event channel to receive and process event data.
+
+Each component can fulfill the role of an event producer, consumer, or both.
+
+As Eve continues her conversation with Chan, she realizes she hasn't offered him a beverage. She quickly rushes to prepare a cup of coffee. While making the coffee, she draws upon a coffee shop as an analogy: a customer (producer) places an order for an espresso (event), conveying the order details to the barista (consumer). The sequence unfolds as follows:
+
+1. The customer initiates an espresso order (an event), and the barista receives the order details.
+2. Subsequently, the barista processes the order and begins crafting the espresso (a response to the event).
+
+In this coffee shop scenario, event-driven architecture mirrors the seamless flow of orders and drinks without any disruption, ensuring a smooth and efficient operation.
+Chan asks Eve how the AsyncAPI specification impacts the event-driven architecture world.
+
+## Event-driven architecture with AsyncAPI
+
+AsyncAPI simplifies the definition of communication and message exchange among different components in an Event-Driven Architecture (EDA). It provides a structured way to describe events, channels, and message formats, enhancing overall understanding.
+
+AsyncAPI helps you define asynchronous APIs. Asynchronous APIs are like digital mailboxes where you can send messages without waiting for an instant reply. When the other person reads your letter and has something to say, they will reply. AsyncAPI maps out events and their responses in an EDA.
```

Comment:
Not sure my suggestion it is well formulated but I would suggest we include something else extra besides just events.

```suggestion
AsyncAPI helps you define asynchronous APIs. Asynchronous APIs are like digital mailboxes where you can send messages without waiting for an instant reply. When the other person reads your letter and has something to say, they will reply. AsyncAPI maps out events, their communication channels, and the responses in an EDA.
```